taskfile/ast/include.go

```diff
@@ -113,6 +113,11 @@ func (includes *Includes) UnmarshalYAML(node *yaml.Node) error {
 			keyNode := node.Content[i]
 			valueNode := node.Content[i+1]
 
+			// Ensure the include value is not null, as it must be either a string or an include object.
+			if valueNode.Kind == yaml.ScalarNode && valueNode.Tag == "!!null" {
```

Comment:
The `null` check is placed before decoding the `valueNode` because the `Decode` method from the yaml package does not invoke the `UnmarshalYAML` method if the node is `null`. This ensures that `null` values are caught early and handled with a proper error message, as the `UnmarshalYAML` logic would be skipped for such nodes.